src/pages/ProvideLiquidity/index.tsx

```diff
@@ -68,30 +86,42 @@ export default function ProvideLiquidity() {
     return <ChangeNetworkModal />
   }
 
+  const cardParams: { title: string; body: string; cta: string; url: string } = showLegacyAddLiquidity
+    ? {
+        title: t('legacyLiquidityProviderCardTitle'),
+        body: t('legacyLiquidityProviderCardBody'),
+        cta: t('legacyLiquidityProviderCardCTA'),
+        url: 'https://revo.market/#/add-liquidity-legacy',
```

Comment:
Whenever I click this, it redirects to the zap page for some reason

Reply:
it is because the change is not live on the deployed instance

Reply:
would be nicer to get the base URL dynamically, but I was hacking

Reply:
:facepalm: of course lol